config/sites/hr.uiowa.edu/core.entity_form_display.taxonomy_term.units.default.yml

```diff
@@ -88,4 +88,18 @@ content:
     region: content
     settings: {  }
     third_party_settings: {  }
-hidden: {  }
+  simple_sitemap:
+    weight: 10
+    region: content
+    settings: {  }
+    third_party_settings: {  }
+  status:
+    type: boolean_checkbox
+    weight: 100
+    region: content
+    settings:
+      display_label: true
+    third_party_settings: {  }
```

Comment:
Just double-checking that this should be here.

Reply:
Hmm. Good eye. I think this is just a really old piece of config that hasn't been touched in a while. It looks like publish and simple sitemap is visible on other taxonomies (e.g. admissions). I'll try to adjust this one as well

Reply:
wait, i misread this diff. this is correct and aligns with other taxonomies.